### PR TITLE
Map layer store

### PR DESCRIPTION
**Pull request by odenypeter**
## What is the Purpose?
- Start o displaying the layers

## What was the approach?
- Implement layer store and start on the sample layers on the map

## Mentions?
@TAnas0 @andrewtpham @ninetteadhikari @Kimaiyo077

### CODE REVIEW

File: src/views/maps/layers.js [hunk 1]
```diff
@@ -3,7 +3,7 @@ export const layers = {
     type: 'fill',
     layout: {},
     paint: {
-      'fill-color': '#EBEBFF',
+      'fill-color': '#ebebff',
```
Comment (ninetteadhikari) [inline src/views/maps/layers.js:6]: instead of HEX we can use `$primary-color` :)

File: src/views/maps/layers.js [hunk 2]
```diff
@@ -3,7 +3,7 @@ export const layers = {
...
       'fill-opacity': 0.5,
       'fill-outline-color': '#3333FF'
```
Comment (ninetteadhikari) [inline src/views/maps/layers.js:8]: we can use this `$primary-fill`

File: src/views/maps/layers.js (hunk 1 above)
Comment (andrewtpham) [reply src/views/maps/layers.js:6]: This is mapbox so I'm not sure we'll be able to use our scss variables

File: src/views/maps/layers.js (hunk 2 above)
Comment (andrewtpham) [reply src/views/maps/layers.js:8]: Same as above ☝️

**Pull request merged**